------Start of DE processing------ = [30/07/25, 08:21:54:395 COT]
Exception = jakarta.resource.spi.ResourceAllocationException
Source = com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection
probeid = 299
Stack Dump = jakarta.resource.spi.ResourceAllocationException: DSRA8100E: No se ha podido obtener PooledConnection de DataSource. with SQL State : 08001 SQL Code : 0
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1604)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1250)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1471)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:576)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:308)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:142)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:244)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:149)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:174)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:359)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:318)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:451)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:219)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:154)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:530)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:824)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:261)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:771)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:221)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getDatabaseSession(EntityManagerFactoryDelegate.java:199)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.getDatabaseSession(EntityManagerFactoryImpl.java:534)
	at org.eclipse.persistence.jpa.PersistenceProvider.createContainerEntityManagerFactoryImpl(PersistenceProvider.java:385)
	at org.eclipse.persistence.jpa.PersistenceProvider.createContainerEntityManagerFactory(PersistenceProvider.java:313)
	at com.ibm.ws.jpa.management.JPAPUnitInfo.createEMFactory(JPAPUnitInfo.java:936)
	at com.ibm.ws.jpa.management.JPAPUnitInfo.initialize(JPAPUnitInfo.java:772)
	at com.ibm.ws.jpa.management.JPAPxmlInfo.extractPersistenceUnits(JPAPxmlInfo.java:184)
	at com.ibm.ws.jpa.management.JPAScopeInfo.processPersistenceUnit(JPAScopeInfo.java:90)
	at com.ibm.ws.jpa.management.JPAApplInfo.addPersistenceUnits(JPAApplInfo.java:121)
	at com.ibm.ws.jpa.container.osgi.internal.JPAComponentImpl.processWebModulePersistenceXml(JPAComponentImpl.java:595)
	at com.ibm.ws.jpa.container.osgi.internal.JPAComponentImpl.applicationStarting(JPAComponentImpl.java:378)
	at com.ibm.ws.container.service.state.internal.ApplicationStateManager.fireStarting(ApplicationStateManager.java:53)
	at com.ibm.ws.container.service.state.internal.StateChangeServiceImpl.fireApplicationStarting(StateChangeServiceImpl.java:52)
	at com.ibm.ws.app.manager.module.internal.SimpleDeployedAppInfoBase.preDeployApp(SimpleDeployedAppInfoBase.java:549)
	at com.ibm.ws.app.manager.module.internal.SimpleDeployedAppInfoBase.installApp(SimpleDeployedAppInfoBase.java:510)
	at com.ibm.ws.app.manager.module.internal.DeployedAppInfoBase.deployApp(DeployedAppInfoBase.java:351)
	at com.ibm.ws.app.manager.war.internal.WARApplicationHandlerImpl.install(WARApplicationHandlerImpl.java:67)
	at com.ibm.ws.app.manager.internal.statemachine.StartAction.execute(StartAction.java:184)
	at com.ibm.ws.app.manager.internal.statemachine.ApplicationStateMachineImpl.enterState(ApplicationStateMachineImpl.java:1369)
	at com.ibm.ws.app.manager.internal.statemachine.ApplicationStateMachineImpl.run(ApplicationStateMachineImpl.java:912)
	at com.ibm.ws.threading.internal.ExecutorServiceImpl$RunnableWrapper.run(ExecutorServiceImpl.java:247)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.sql.SQLNonTransientException: Public Key Retrieval is not allowed DSRA0010E: Estado SQL = 08001, CÃ³digo de error = 0
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:102)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:833)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:416)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:238)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:180)
	at com.mysql.cj.jdbc.MysqlDataSource.getConnection(MysqlDataSource.java:408)
	at com.mysql.cj.jdbc.MysqlDataSource.getConnection(MysqlDataSource.java:120)
	at com.mysql.cj.jdbc.MysqlConnectionPoolDataSource.getPooledConnection(MysqlConnectionPoolDataSource.java:58)
	at com.ibm.ws.rsadapter.impl.DatabaseHelper$1.run(DatabaseHelper.java:1518)
	at com.ibm.ws.rsadapter.impl.DatabaseHelper$1.run(DatabaseHelper.java:1502)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.base/java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:618)
	at com.ibm.ws.rsadapter.impl.DatabaseHelper.getPooledConnection(DatabaseHelper.java:1502)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:857)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:698)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1376)
	... 42 more
Caused by: com.mysql.cj.exceptions.UnableToConnectException: Public Key Retrieval is not allowed
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:52)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:76)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:122)
	at com.mysql.cj.protocol.a.authentication.CachingSha2PasswordPlugin.nextAuthenticationStep(CachingSha2PasswordPlugin.java:40)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.proceedHandshakeWithPluggableAuthentication(NativeAuthenticationProvider.java:436)
	at com.mysql.cj.protocol.a.NativeAuthenticationProvider.connect(NativeAuthenticationProvider.java:206)
	at com.mysql.cj.protocol.a.NativeProtocol.connect(NativeProtocol.java:1333)
	at com.mysql.cj.NativeSession.connect(NativeSession.java:155)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:957)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:821)
	... 56 more

Dump of callerThis
Object type = com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource
  strings[0] = "_______________________________________________________________________"
  strings[1] = ""
  strings[2] = "    First Failure Data Capture information for"
  strings[3] = "          com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource@6cc33347"
  strings[4] = "_______________________________________________________________________"
  strings[5] = ""
  strings[6] = "ResourceRefInfo:"
  strings[7] = "  
[Resource-ref CMConfigData key items]

	res-sharing-scope:        0 (SHAREABLE)
	res-isolation-level:      0 (TRANSACTION_NONE)
	res-auth:                 1 (APPLICATION)
	commitPriority            0
	branchCoupling            -1
	loginConfigurationName:   null
	loginConfigProperties:    {}
	Resource ref name:        not set
	Queue manager id:        not set

"
  strings[8] = "Connection Manager:"
  strings[9] = "  [ConnectionManager]@1a511920
JNDI Name <jdbc/tuappDS>
shareable <true>

"
  strings[10] = "_______________________________________________________________________"
  strings[11] = ""
  strings[12] = "    First Failure Data Capture information for"
  strings[13] = "          com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl@b4c36bc"
  strings[14] = "_______________________________________________________________________"
  strings[15] = ""
  strings[16] = "com.ibm.ws.rsadapter.DSConfig@3fd5632"
  strings[17] = "[beginTranForResultSetScrollingAPIs, true, beginTranForVendorAPIs, true, commitOrRollbackOnCleanup, null, connectionSharing, MatchOriginalRequest, isolationLevel, -1, jndiName, jdbc/tuappDS, enableConnectionCasting, false, queryTimeout, null, statementCacheSize, 10, supplementalJDBCTrace, null, syncQueryTimeoutWithTransactionTimeout, false, transactional, true]"
  strings[18] = "{password=******, databaseName=overtime, allowPublicKeyRetrieval=false, serverName=localhost, serverTimezone=UTC, user=overtimeuser, portNumber=3306, useSSL=false}"
  strings[19] = "Vendor implementation class:"
  strings[20] = "  class com.mysql.cj.jdbc.MysqlConnectionPoolDataSource
"
  strings[21] = "Type:"
  strings[22] = "  interface javax.sql.ConnectionPoolDataSource
"
  strings[23] = "Underlying DataSource or Driver Object: com.mysql.cj.jdbc.MysqlConnectionPoolDataSource@6b1d110"
  strings[24] = "  com.mysql.cj.jdbc.MysqlConnectionPoolDataSource@6b1d110
"
  strings[25] = "Instance id:"
  strings[26] = "  1
"
  strings[27] = "Log Writer:"
  strings[28] = "  null
"
  strings[29] = "Counter of fatal connection errors on ManagedConnections created by this MCF:"
  strings[30] = "  0
"

